modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -190,6 +190,48 @@ Parameters for short form options will be written either directly after the opti
 sudo -u www-data ./occ files:scan -p "user_x/files/folder"  
 ....
 
+[[Brute Force Protection-commands]]
+Brute Force Protection Commands
+-------------------------------
+
+Use these commands to configure the Brute Force Protection app.
+Parametrisation must be done with the `occ config` command set.
+The combination of `uid` and ÌP address` is used to trigger the ban.
+
+List the current settings
```

Comment:
Please use [title case](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm) for all headers